### ISSUE TITLE
Feature: Adding comment when saving new version

### ISSUE THREAD

**Issue by bhavincb** [NativeScript/playground-feedback#74]
while saving the new version for some project if it has an option to add some message or comment then it can be very usefull to give insight to old saves and what changes has been made from that point of time.

**Comment by vhristov5555** [comment]
Hi,

In the moment you can change the title of the playground which is visible in the projects view:
![image](https://user-images.githubusercontent.com/7846198/44772601-f6359f00-ab76-11e8-94d8-d1a5789198e6.png)
![image](https://user-images.githubusercontent.com/7846198/44772710-4ad91a00-ab77-11e8-952c-cce892684b0e.png)

Hope this helps.

Best Regards,
Vasil Hristov

**Comment by bhavincb** [comment]
yes, currently i'm already using that workaround. but it will be more useful and meaningful if there is comment option.

**Comment by vhristov5555** [comment]
We are going to add this feature to our backlog. 
Thanks for the feedback.
